### ISSUE TITLE
Is this the best? :)

### ISSUE THREAD

**Issue by apawsey** [niieani/aurelia-async-binding-function#1]
Trying to find a way to work around a promise binding, and as far as I can see, this seems to be the best solution at the moment?  Trying not to be too biased, does that sound correct or would you recommend a different approach?

**Comment by niieani** [comment]
@apawsey Actually the least obtrusive way to bind to Promises is to wrap them like this:
https://gist.run/?id=bd1ee888a9311027156b29b7df6ac27e

The plugin here introduces custom syntax `@` and as far as I saw that didn't work well in repeaters. 
I might drop the `@` and just make it so that the binding functions don't load globally. Then this plugin would be a nicer solution than the one cited above.

Cheers.

**Comment by istrau2** [comment]
@niieani what about this approach: https://gist.run/?id=524b94cc960456499bbb306fe1491e4a

**Comment by niieani** [comment]
@istrau2 yeah, definitely looks like a good option. I don't recommend using binding-function's anymore.
